index.js

```diff
@@ -52,4 +53,135 @@ const state = {
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+
+// SELECTORS
+const storeItemList = document.querySelector('.store--item-list')
+const cartItemList = document.querySelector('.cart--item-list')
+const totalNumber = document.querySelector('.total-number')
+
+
+const clearStorePage = () => {
+  storeItemList.innerHTML = ""
+}
+
+const clearCart = () => {
+  cartItemList.innerHTML = ""
+}
+
+const renderAllTheItemsInTheStore = () => {
+  clearStorePage()
+  state.items.forEach(currentItem => {
```

Comment:
Happy to see that you went back and changed from using a generic for loop to the using a for each array function. In my option it results in a more readable code with less points of potential issues.